CHANGELOG.md

```diff
@@ -84,6 +84,7 @@ Make sure to upgrade to v0.25.0 so that your node will continue to connect once
 - `cli generateWallet` renamed to `cli walletCreate`
 - `cli generateAddresses` renamed to `cli walletAddAddresses`
 - `/api/v1/explorer/address` is deprecated in favor of `/api/v1/transactions?verbose=1`
+- Switch `skycoin-cli` from `urfave/cli` to `spf13/cobra`
```

Comment:
This requires all options to use `--` instead of `-` right?

Reply:
yes